screens/Home/index.tsx

```diff
@@ -7,13 +7,16 @@ import Screen from '../../components/Screen'
 import { Text } from '../../components/Themed'
 import Colors from '../../constants/Colors'
 import { meditations, MeditationItem } from '../../data/meditations'
+import useColorScheme from '../../hooks/useColorScheme'
 import { HomeParamList } from '../../types'
 
 interface Props {
   navigation: StackNavigationProp<HomeParamList, 'HomeScreen'>
 }
 
 export default function Home({ navigation }: Props) {
+  const colorScheme = useColorScheme()
```

Comment:
I wouldn't do it this way, its a bit inconsistent with the rest of the codebase and there is a simpler approach. Instead use `useThemeColor` here and then you don't need all this extra logic below. I'll explain further in my other comments.

```suggestion
  const textColor = useThemeColor({}, 'text')
```